FeedStoreChallenge/Infrastructure/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by GD on 9/2/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
This class is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```